### DISCUSSION TITLE
Manager says "Lsposed not installed"

### DISCUSSION THREAD

**Discussion by gokuta2** [LSPosed/LSPosed#2037] [General]
My phone is Motorola Moto G6, OS Android 9 (up to date).
I have installed Magisk V25.1 with Zygisk enabled. All Magisk modules work well, but not LSposed.

Magisk says I have LSposed v1.8.3 installed (and enabled). Shortcut of manager does not appear on desktop after reboot. I navigate to `/data/adb/lspd/manager.apk`, but `manager.apk` fails in launch with a message "there was a problem parsing the package". Dialing the number `*#*#5776733#*#*` does nothing. I can install `manager.apk` though when moving it to a different location. So, I moved it to default Downoads location and installed it. I open LSposed manager and get this message "LSposed no installed". Then I installed LSposed `Disable FLAG_SECURE` module so I could take screenshots of anything I want. And yes I cannot take screenshots, module does not work.

LSposed does not work. I can see other people are having this exact same issue. I would like some help with this, should I send you logs (where are they?) or what should I do? I am pretty sure installing LSposed is not rocket science. If it was actually working, I would have installed it already.

**Comment by yujincheng08** [comment]
send log from `/data/adb/lspd/log`

**Reply by gokuta2** [reply]
No such directory, `manager.apk` is all I see in there

**Reply by yujincheng08** [reply]
Then we need a full `adb logcat` during boot.

**Comment by gokuta2** [comment]
[logcat.zip](https://github.com/LSPosed/LSPosed/files/9127505/logcat.zip)

I turned my phone off, opened command-line and typed in `adb -d logcat > logcat.txt`. Then I turn on my phone and for a while nothing happended. Then Android has finished loading and logging has begun. I waited for ~3min and then stopped logging. The result is attached as `logcat.zip`

**Reply by yujincheng08** [reply]
See #785. We cannot fix it.

**Reply by gokuta2** [reply]
okay, then I give up. And thank you for help

**Reply by mywalkb** [reply]
why do you give up? There are 2 solutions [1](https://github.com/LSPosed/LSPosed/issues/785#issuecomment-869190719) [2](https://github.com/LSPosed/LSPosed/issues/785#issuecomment-869196215) 
[Here](https://github.com/LSPosed/LSPosed/discussions/1988#discussioncomment-2951061) a user solved it with first solution
